### DISCUSSION TITLE
Contours

### DISCUSSION THREAD

**Discussion by lejsekt** [SixLabors/ImageSharp#1653] [Q&A] [answered]
Hi,
Is there something in image sharp like open cv contours (or "regions")?

I was wondering if it's possible to compute a bounding box of a white blob in an otherwise black image (mask).

Thank you

**Comment by antonfirsov** [comment]
Can you share some links or more detailed description about this feature?

From what I understand, the closest thing related to this is [Entropy Crop](https://docs.sixlabors.com/api/ImageSharp/SixLabors.ImageSharp.Processing.EntropyCropExtensions.html),  maybe it can be parameterized to extract "a white blob in an otherwise black image", but not sure.

**Reply by JimBobSquarePants** [reply]
We've had a couple of people ask for something like this before. EntropyCrop would return a cropped image representing those bounds but there's no way to just get the bounds. My concern previously is how to expose a non mutating API in a generic way.

**Comment by lejsekt** [comment, marked as answer]
By contours I mean something like this: https://docs.opencv.org/4.5.2/d4/d73/tutorial_py_contours_begin.html

**Reply by antonfirsov** [reply]
Thanks! This is clearly a Computer Vision concern, which is out of scope for both ImageSharp and ImageSharpp.Drawing.

That said, I would enjoy seeing someone building a fully managed CV library.
